packages/nuxt/src/generators/application/files/base/nuxt.config.ts__tmpl__

```diff
@@ -0,0 +1,31 @@
+import viteTsConfigPaths from 'vite-tsconfig-paths';
```

Comment:
Wouldn't it be better to use the one from Nx? We add an unneeded @nx/vite dep, yes, but the Vite plugin from the Nx package utilizes the Nx graph instead of typescripts tsconfig

Reply:
I took the latest approach from the @nx/web:app generator and it looks like they switched from `nxViteTsPaths` to `npm:vite-tsconfig-paths`

Reply:
Mh, where? afaik they are just switching everything to `nxViteTsPaths`

Reply:
Ooops, yeah, I just checked again and it's the other way around. Earlier "@nx/vite" versions used the vite-tsconfig-path. I'll change that now, thanks!